client/my-sites/site-settings/form-discussion.jsx

```diff
@@ -179,6 +179,12 @@ class SiteSettingsFormDiscussion extends Component {
 					onChange={ this.handleCommentOrder }>
 					<span>{ translate( 'Comments should be displayed with the older comments at the top of each page' ) }</span>
 				</CompactFormToggle>
+				<CompactFormToggle
+					checked={ fields[ 'gravatar-hovercards' ] }
```

Comment:
Dot-notation couldn't be used because the property contains `-`, so it's not a valid identifier 😢 . It's a bool so probably no need for explicit conversion, but this all becomes moot as I change the toggle to `JetpackModuleToggle`.